src/main/scala/firrtl/Emitter.scala

```diff
@@ -545,17 +545,19 @@ class VerilogEmitter extends SeqTransform with Emitter {
       this(Seq(), Map.empty, m, moduleMap, "", new EmissionOptions(Seq.empty))(writer)
     }
 
-    val netlist = mutable.LinkedHashMap[WrappedExpression, Expression]()
+    val netlist = mutable.LinkedHashMap[WrappedExpression, (Expression, Info)]()
```

Comment:
I'm perfectly fine with tuples, but the choice of a tuple here is different from the modified transforms that use `InfoExpr`. Can you elaborate on your reasoning for using a tuple here and a case class elsewhere?

Reply:
It predates `InfoExpr`, might as well use `InfoExpr`, I'll change it.